<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiberalMachOption</class>
 <widget class="QMainWindow" name="LiberalMachOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transparent;
background-color: rgb(255, 208, 142);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>150</y>
      <width>291</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_6_0">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ICT와 경영</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_1">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>공학도의 철학메모</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_2">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>디자인과 감성적사고</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_3">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>생활 속의 ICT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_4">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>스토리텔링의 이해</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_5">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>이미지인문학</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_6">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>인지행동신경과학</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>149</y>
      <width>333</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBox_6_7">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>창의적 디자인 사고와 비즈니스 모델</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_8">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>창의적 문화분석 입문</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_9">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>초연결사회의 비즈니스모델</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_10">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>MACH포커스</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_11">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>공학인을 위한 품질 경영</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_12">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>미래기술과 도시</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_13">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>인문학적 사고와 스마트 창업</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6_14">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>공감의 심리학</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_right_arrow">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_right_arrow{
background-color: transparent;
border-image: url(:right_arrow3.png);
background: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_main_liberal_ment">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>109</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string>이미 수강하신 MACH교양을 선택하세요. </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_main_liberal_option_title">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>MACH교양</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_borderline">
    <property name="geometry">
     <rect>
      <x>-19</x>
      <y>-30</y>
      <width>897</width>
      <height>919</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/borderline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>620</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/bear2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_borderline</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>pushButton_right_arrow</zorder>
   <zorder>label_main_liberal_ment</zorder>
   <zorder>label_main_liberal_option_title</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="menu_logo.qrc"/>
 </resources>
 <connections/>
</ui>
